index.html

```diff
@@ -0,0 +1,202 @@
+                        <h3 class="services__item-title">
+                            Feel Like A Star
+                        </h3>
+                        <p class="services__item-about">
+                            Quisque hendrerit purus dapibus, ornare nibh vitae, viverra nibh. Fusce vitae aliquam
+                            tellus.
+                        </p>
+                    </div>
+                </article>
+            </div>
+        </div>
+    </section>
+    <div class="decoration-line decoration-line--white"></div>
+    <section class="portfolio" id="portfolio">
+        <div class="wrapper">
+            <h2 class="title">
+                Portfolio
+            </h2>
+            <p class="description">Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio
+                sem nec elit.
+            </p>
+            <div class="portfolio__tabs">
+                <a href="#" class="portfolio__tab portfolio__tab--active">All</a>
+                <a href="#" class="portfolio__tab">Web Design</a>
+                <a href="#" class="portfolio__tab">Graphic Design</a>
+                <a href="#" class="portfolio__tab">Artwork</a>
+            </div>
+            <div class="portfolio__pictures">
+                <img src="assets/images/portfolio-1.png" alt="picture1" class="portfolio__picture">
+                <img src="assets/images/portfolio-2.png" alt="picture2" class="portfolio__picture">
+                <img src="assets/images/portfolio-3.png" alt="picture3" class="portfolio__picture">
+                <img src="assets/images/portfolio-4.png" alt="picture4" class="portfolio__picture">
+                <img src="assets/images/portfolio-5.png" alt="picture5" class="portfolio__picture">
+                <img src="assets/images/portfolio-6.png" alt="picture6" class="portfolio__picture">
+                <img src="assets/images/portfolio-7.png" alt="picture7" class="portfolio__picture">
+                <img src="assets/images/portfolio-8.png" alt="picture8" class="portfolio__picture">
+                <img src="assets/images/portfolio-9.png" alt="picture9" class="portfolio__picture">
+                <img src="assets/images/portfolio-10.png" alt="picture10" class="portfolio__picture">
+                <img src="assets/images/portfolio-11.png" alt="picture11" class="portfolio__picture">
+                <img src="assets/images/portfolio-12.png" alt="picture12" class="portfolio__picture">
+            </div>
+        </div>
+    </section>
+    <div class="decoration-line"></div>
+    <footer class="footer">
+        <p class="copyright">
+            &copy;Copyright 2013
+        </p>
+        <div class="social">
+            <div class="social__circle"> <a href="https://www.facebook.com/" class="social__link" target="_blank"><img
+                        src="assets/images/ico-facebook--nocircle.png" alt="facebook" class="social__icon"></a>
```

Comment:
Format code here: check spaces between tags, don't be afraid to place tags to a new line.